pkg/policy/api/entity.go

```diff
@@ -41,6 +41,8 @@ const (
 
 	// EntityInit is an entity that represents an initializing endpoint
 	EntityInit Entity = "init"
+
+	EntityNone Entity = "none"
```

Comment:
Should we allow this entity to be provided via the API? I think we could allow policy which uses it, but not for labels themselves for endpoints. I.e., we could never provide an endpoint with labels `reserved:none`.

Reply:
That seems reasonable to me. One decision to make around this is whether this becomes the canonical way to express deny-all. It isn't critical to fixing the toFQDN bug, but introducing entityNone and then using `rule.toSomething == nil` and `len(rule.toSomething) = 0` to each mean  something might cause us trouble later.
Merging something like this PR allows the toGroups handling to change slightly, where it would be a deny-all when cilium-agent sees the parent policy, and the derivative policies whitelist the appropriate IPs. The current toGroups behaviour is to ignore the parent policy rules with toGroups outright, only beginning enforcement once a derivative policy is created.